examples/job_crud.py

```diff
@@ -53,7 +55,18 @@ def create_job(api_instance, job):
     api_response = api_instance.create_namespaced_job(
         body=job,
         namespace="default")
-    print("Job created. status='%s'" % str(api_response.status))
+    print("Job created. status='%s'" % str(get_job_status(api_instance)))
```

Comment:
better to keep the previous print line 56 to show the Job is created, then poll for job running status until job is completed

Reply:
@yliaog sleep for 1 sec is good?

Reply:
that depends on the how long it takes to complete the job, 1s is probably ok.

Reply:
@yliaog Please check the code changes now. Thanks.